devops-tools/terraform/README.md

```diff
@@ -22,3 +22,10 @@ The credential of the service account will be printed.
 
 > Note that the credential is only printed for demonstration purposes.
 > Service credentials must be kept secure and should not be logged.
+
+Moreover, a secret named `test` is also created in the specified repo.
+
+To verify that the service indeed has read access on repo, try reading the test secret with the service credential:
+```
+SECRETHUB_CREDENTIAL=<your-service-credential> secrethub read <your-username>/<repo>/test
```

Comment:
Or even easier:
```
SECRETHUB_CREDENTIAL=$(terraform output service_credential) secrethub read $(terraform output secret_path)
```

But this would require us to add an extra output.